test/unit/ServerConfigParams.test.ts

```diff
@@ -1,122 +1,156 @@
 import {
   verifyServerConfiguration,
-  AppConfig,
-  ContractsConfig,
   ServerConfigParams,
-  ERROR_DISABLE_SPONSOR_TX_NOT_CONFIGURED,
   ERROR_GAS_FEE_PERCENTAGE_NEGATIVE,
   ERROR_FIXED_USD_FEE_NEGATIVE,
 } from '../../src/ServerConfigParams';
-import { ethers } from 'ethers';
+import { Wallet } from 'ethers';
 import { expect, use } from 'chai';
 import chaiAsPromised from 'chai-as-promised';
+import config from 'config';
 
 use(chaiAsPromised);
 
 describe('ServerConfigParams tests', function () {
+  //Not all cases are tested here since the validation is made using an external library and it makes no sense
+  //to test the library here. The test are focused in the custom functions of /serverConfigParamsUtils.ts
+  //and the conditional validations in /ServerConfigParams.ts
   describe('Function verifyServerConfiguration()', function () {
-    let contracts: ContractsConfig;
-    let app: AppConfig;
-    let randomAddress: string;
+    let originalConfig: ServerConfigParams;
 
     before(function () {
-      randomAddress = ethers.Wallet.createRandom().address;
+      originalConfig = config.util.toObject(config) as ServerConfigParams;
     });
 
-    beforeEach(function () {
-      app = {
-        url: 'https://fake.rsk',
-        port: 8080,
-        workdir: '/some/dir',
-      } as AppConfig;
+    afterEach(function () {
+      config.util.extendDeep(config, originalConfig);
+    });
+
+    it('Should pass with default configuration', function () {
+      config.util.extendDeep(config, { app: { disableSponsoredTx: false } });
 
-      contracts = {
-        relayHubAddress: randomAddress,
-      } as ContractsConfig;
+      expect(() => verifyServerConfiguration()).not.to.throw();
     });
 
     it('Should fail if disableSponsoredTx is not properly configured', function () {
```

Comment:
```suggestion
    it('Should fail if disableSponsoredTx is not configured', function () {
```

I dont understand what do you mean with properly.

Reply:
Changed.